### PR TITLE
front typed imports

### PR DESCRIPTION
**Pull request by anisometropie**

### CODE REVIEW

File: content/docs/guides/contribute/conventions/front.en.md [hunk 1]
```diff
@@ -73,3 +73,32 @@ Utiliser les endpoints générés à partir des fichiers `openapi.yaml` pour con
...
+- Avoid dependency cycles:
+
+ ![dependency cyle](./dependency-cycle.png)
+
+The error disappears with the `type` keyword
+
+ ![dependency cyle](./dependency-cycle-gone.png)
+
+
+- make final bundle lighter (all types disappear at compilation)
```
Comment (axrolld) [inline content/docs/guides/contribute/conventions/front.en.md:104]: Why is the text above this part in French ?

Comment (axrolld) [inline content/docs/guides/contribute/conventions/front.en.md:104]:
```suggestion
- Make final bundle lighter (all types disappear at compilation).
```

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 2]
```diff
@@ -73,3 +73,31 @@ Utiliser les endpoints générés à partir des fichiers `openapi.yaml` pour con
...
+export type { Direction, DirectionalTrackRange as TrackRange };
+```
+```typescript
+import type { typedEntries, ValueOf } from 'utils/types';
+```
+
+Cette pratique permet de&nbsp;:
+- d’améliorer les performances et le travail d’analyse du compilateur et du linter.
+- Rendre ces déclarations plus lisible, on voit clairement ce qu’on est en train d’importer.
+- Éviter des cycles de dépendances :
```
Comment (axrolld) [inline content/docs/guides/contribute/conventions/front.fr.md:94]:
```suggestion
- Améliorer les performances et le travail d’analyse du compilateur et du linter.
- Rendre ces déclarations plus lisibles, on voit clairement ce qu’on est en train d’importer.
- Éviter des cycles de dépendances :
```

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 3]
```diff
@@ -73,3 +73,31 @@ Utiliser les endpoints générés à partir des fichiers `openapi.yaml` pour con
...
+
+ ![dependency cyle](./dependency-cycle.png)
+
+l’erreur disparaît avec le mot clé `type`
```
Comment (axrolld) [inline content/docs/guides/contribute/conventions/front.fr.md:98]: To match with the english version
```suggestion
L’erreur disparaît avec le mot clé `type`
```

File: content/docs/guides/contribute/conventions/front.fr.md [hunk 4]
```diff
@@ -73,3 +73,31 @@ Utiliser les endpoints générés à partir des fichiers `openapi.yaml` pour con
...
+
+ ![dependency cyle](./dependency-cycle-gone.png)
+
+
+- Alléger le bundle final (tous les types disparaissent à la compilation)
```
Comment (axrolld) [inline content/docs/guides/contribute/conventions/front.fr.md:103]:
```suggestion
- Alléger le bundle final (tous les types disparaissent à la compilation).
```

**Review by axrolld — CHANGES_REQUESTED** [review]
A few fixes about uppercase and punctuation needed

File: content/docs/guides/contribute/conventions/front.en.md (hunk 1 above)
Comment (axrolld) [reply content/docs/guides/contribute/conventions/front.en.md:104]: This is not related specifically to your PR, i was just wondering

Comment (anisometropie) [reply content/docs/guides/contribute/conventions/front.en.md:104]: Yeah, the translations weren’t done for the rest

**Review by axrolld — APPROVED** [review]

**Review by nicolaswurtz — APPROVED** [review]

**Pull request merged**